go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/golangci/golangci-lint
 
-go 1.12
+go 1.13
```

Comment:
Really upgrade go version?

Reply:
- the current min go version in the GitHub Action is go1.13
- in the code, the `%w` is used (available since go1.13)